RecoJets/JetProducers/interface/HTTTopJetTagInfo.h

```diff
@@ -0,0 +1,98 @@
+#ifndef DataFormats_JetReco_interface_HTTTopJetTagInfo_h
```

Comment:
what is this file (RecoJets/JetProducers/interface/HTTTopJetTagInfo.h) doing here?
It appears identical to DataFormats/JetReco/interface/HTTTopJetTagInfo.h
even the include guard is the same